devdox/app/repositories/repo_repository.py

```
@@ -18,3 +18,22 @@ async def get_all_by_user(
 
     async def count_by_user(self, user_id: str) -> int:
         return await Repo.filter(user_id=user_id).count()
+
+    async def create_new_repo(self, user_id, token_id, repo_id, repo_name, description, html_url, default_branch,
```

Pylint (3.3.7)

[refactor] Too many arguments (15/5)

(R0913)